src/main/java/com/google/devtools/build/lib/packages/RepoThreadContext.java

```diff
@@ -14,18 +14,26 @@
 
 package com.google.devtools.build.lib.packages;
 
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
 import com.google.devtools.build.lib.cmdline.RepositoryMapping;
 import com.google.devtools.build.lib.cmdline.StarlarkThreadContext;
+import java.util.Collection;
+import java.util.Map;
 import net.starlark.java.eval.EvalException;
 import net.starlark.java.eval.Starlark;
 import net.starlark.java.eval.StarlarkThread;
 
 /** Context object for a Starlark thread evaluating the REPO.bazel file. */
 public class RepoThreadContext extends StarlarkThreadContext {
   private final LabelConverter labelConverter;
```

Comment:
this isn't used anymore right? we should remove it if so.

Reply:
double checking: is this done?

Reply:
Now it is, sorry. For some reason I thought that you meant unused import statements, saw nothing gray in the IDE and called it good.